src/backend.py

```diff
@@ -4,6 +4,14 @@
 class ParadoxClient:
     def __init__(self, http_client):
         self.http_client = http_client
+        self._skus = {}
+
+    async def get_skus(self):
+        if not self._skus:
+            skus = await self.http_client.do_request('GET', 'https://accounts.paradoxplaza.com/api/skus',
+                                                     headers={'content-type': 'application/json'})
+            self._skus = await skus.json()
+        return self._skus

```

Comment:
I'm not sure if this is the best way to do this, but I thought it'd be good to buffer this since it's just a list of all the games on Paradox.